apps/conte/src/app/Shared/components/modals/generic/generic-modal.component.html

```diff
@@ -0,0 +1,96 @@
+<!-- Header -->
+<div class="g-modal-content">
+  <div class="g-modal-header">
+    <h4 *ngIf="heading" class="modal-title display-5">{{ heading }}</h4>
+    <h4 *ngIf="subHeading" class="modal-title display-6">{{ subHeading }}</h4>
+    <div *ngIf="logo" class="d-flex justify-content-center">{{ logo }}</div>
+  </div>
+
+  <div class="g-modal-body">
+    <div class="d-flex justify-content-center mb-3">{{ body }}</div>
+
+    <!-- List -->
+    <div *ngIf="list">
+      <div *ngFor="let item of list" class="list mt-1">
+        <li>{{ item }}</li>
+        <button *ngIf="listSecAction" (click)="listSecAction(item, list)" class="list-btn">
+          {{ listSecActionText }}<i *ngIf="listSecActionLogo === 'skip'" class="bi bi-skip-forward ms-1"></i>
+        </button>
+        <button *ngIf="listAction" (click)="listAction(item)" class="list-btn">
+          {{ listActionText }}<i *ngIf="listActionLogo === 'navigate'" class="bi bi-arrow-bar-right ms-1"></i>
+        </button>
+      </div>
+    </div>
+
+    <!-- Form -->
+    <div *ngIf="form" [@fade]>
+      <form (ngSubmit)="onSubmit()">
+        <div>
+          <div class="form-group">
+            <label for="exampleInputEmail1">Title</label>
+            <br />
+            <input class="form-control" [(ngModel)]="form" [ngModelOptions]="{ standalone: true }" />
```

Comment:
are we sure this input field should be using 'form' as a model?

Reply:
that is how we get value in input field